### PR TITLE
[SITES-513] - Calendar callout visual update

### PR DESCRIPTION
**Pull request by simonschwartz**
The PR updates the visual design of the calendar callout used on the times and dates section

### CODE REVIEW

File: app/assets/stylesheets/_callout-basic.scss [hunk 1]
```diff
@@ -1,9 +1,19 @@
...
+
+.callout--calendar-event {
+  box-shadow: none;
+  margin-left: 0;
+  margin-right: 0;
+
+  .next-event {
+    margin-top: $small-spacing;
+  }
+}
```
Comment (tessereth) [inline app/assets/stylesheets/_callout-basic.scss]: Are we overriding ui-kit here? Is this important enough to do that?

**Comment by klepas** [comment]
Soz on the the back-and-forth on this.

I got confirmation from Gaz that the box shadow was fine when I added this initially.

I’ve mentioned this to Alz and we’ll likely huddle up in the coming week or two to unify our callout styles properly.

**Comment by simonschwartz** [comment]
Looks like we can pause this feature for the moment. Any objections if I close this now?

**Comment by klepas** [comment]
None from me — I like the idea of smashing these all out in one go in the coming week or two with Alz, and saving you from making these edits, and then someone undoing them later.

**Pull request closed without merging**